Spec/RestClient.swift

```diff
@@ -646,6 +646,39 @@ class RestClient: QuickSpec {
 
                     expect(NSRegularExpression.match(capturedURLs[1], pattern: "//[f-j].ably-realtime.com")).to(beTrue())
                 }
+
+                // RSC15b
+                it("applies when ClientOptions#fallbackHostsUseDefault is true") {
+                    let options = ARTClientOptions(key: "xxxx:xxxx")
```

Comment:
`fallbackHostsUseDefault` being true does nothing unless a custom host / environment is being used.  You need to specify a custom env / host as well as `fallbackHostsUseDefault`